### DISCUSSION TITLE
Your free GitHub Copilot access has expired

### DISCUSSION THREAD

**Discussion by shahadat3031diu** [community/community#53997] [Copilot Conversations] [answered]
### Select Topic Area

Question

### Body

Student account, but not getting the free service. Can anyone help me with this? Around 21 days back I received this email.
![Screenshot_3](https://user-images.githubusercontent.com/79907609/235238661-a7d20475-2261-4a86-9508-d4b41fc12a90.png)

**Comment by davecheney** [comment]
@shahadat3031diu there is no active student coupon on your account. If you are a student, once you complete the copilot for education verification process you will be provided with a coupon code you can apply to your account. Once that occurs, you can sign up for a student copilot account.

**Reply by shahadat3031diu** [reply]
I've used this feature since last year as a student by my university email (.edu.bd domain). I don't know why this goes out. Could you please guide me?

**Reply by davecheney** [reply, marked as answer]
You'll find the details here https://education.github.com/pack
